packages/settingeditor/test/settingsformeditor.spec.tsx

```diff
@@ -114,10 +123,16 @@ describe('@jupyterlab/settingeditor', () => {
       // Then
       const instance = component.root;
       const form = instance.findByType(FormComponent);
-      expect(form.props.formData).toEqual({ keyA: 'A', keyC: 'C' });
+      expect(form.props.formData).toEqual({
+        keyA: 'A',
+        keyC: 'C',
+        keyComma: ','
+      });
       expect(Object.keys(form.props.schema.properties)).toEqual(
         Object.keys(form.props.formData)
       );
+      //check AriaLabels are
+      expect(ariaLabels).toEqual(form.props.formData.ariaLabel);
```

Comment:
I do not follow this test. It appears that you are testing the generic settings editor, but you did not make any changes to source code in `settingeditor/src`.

If you wanted to test `shortcuts-extension` components I would suggest going the same route as I went in https://github.com/jupyterlab/jupyterlab/pull/16043/files#diff-ef23219df9ce6e7419379a5189c9252aac220bdf696d407f45d99a0c33ab453e

Reply:
This is creating a new e.g. `shortcuts-extension/test/components/ShortcutInput.spec.ts` file mirroring the shortcuts-extension/test/components/ShortcutUI.spec.ts` file from my PR. Of note it might be best deferred until that PR is merged as it simplifies testing individual components by reducing the external bundle content.